acmw-db-app/pages/reset.js

```diff
@@ -0,0 +1,176 @@
+import Head from 'next/head'
+import styles from '../styles/Reset.module.css'
+import React, { useState } from 'react';
+  
+class ResetPW extends React.Component {
+    constructor() {
+    super();
+    this.state = {
```

Comment:
I think Sara suggested for me to use hooks, and tbh I don't know how much more advantageous it is to use one or the other, but I'm just gonna copy over the two articles she shared:
1. https://reactjs.org/docs/hooks-intro.html
2. https://medium.com/@Zwenza/functional-vs-class-components-in-react-231e3fbd7108

Reply:
Hooks can definitely be a bit tricky to understand but it avoids state and generally makes your code shorter/cleaner.